communication/slack-guidelines.md

```diff
@@ -1,90 +1,212 @@
 
-Slack Admins should make sure to mention this in the “What I do” section of their Slack profile, as well as for which time zone.
+## Admins
 
-To connect: please reach out in the #slack-admins channel, mention one of us in the specific channel where you have a question, or DM (Direct Message) one of us privately.
+- Check the [centralized list of administrators][admins] for contact information.
 
-### ADMIN EXPECTATIONS AND GUIDELINES
-* Adhere to Code of Conduct
-* Take care of spam as soon as possible, which may mean taking action by making members inactive
-* Moderating and fostering a safe environment for conversations
-* Bring Code of Conduct issues to the Steering Committee
-* Create relevant channels and list Code of Conduct in new channel welcome message
-* Help troubleshoot Slack issues
-* Review bot, token, and webhook requests
-* Be helpful!
+Slack Admins will have listed that they are a Slack admin in their Slack profile,
+along with their specific timezone.
 
-## CREATING CHANNELS
-Please reach out to the #slack-admins group with your request to create a new channel.
+To connect: please reach out in the `#slack-admins` channel, mention an admin
+directly in the `#slack-admins` channel when you have a question, or DM (Direct
+Message) one privately.
 
-Channels are dedicated to [SIGs, WGs](/sig-list.md), sub-projects, community topics, and related Kubernetes programs/projects.  
-Channels are not:
-* company specific; cloud providers are ok with product names as the channel. Discourse will be about Kubernetes-related topics and not proprietary information of the provider.
-* private unless there is an exception: code of conduct matters, mentoring, security/vulnerabilities, github management, or steering committee.  
+---
 
-Typical naming conventions:
-#kubernetes-foo #sig-foo #meetup-foo #location-users #projectname
+## General Communication Guidelines
 
-All channels need a documented purpose. Use this space to welcome the targeted community: promote your meetings, post agendas, etc.
+### Workspace Channel History
 
-We may make special accommodations where necessary.
+The Kubernetes Slack Workspace is archived and made available when the
+administrators have time. There is no explicit interval.
+[The latest archive is from June 2016-November 2018].
 
-## ESCALATING and/or REPORTING A PROBLEM
-Join the #slack-admins channel or contact one of the admins in the closest timezone via DM directly and describe the situation. If the issue can be documented, please take a screenshot to include in your message.
 
-What if you have a problem with an admin?
-Send a DM to another listed Admin and describe the situation OR
-If it’s a code of conduct issue, please send an email to conduct@kubernetes.io and describe the situation
+### Sending Messages to the Channel
 
-## BOTS, TOKENS, WEBHOOKS, OH MY
+`@here` and `@channel` should be used rarely. Members will receive notifications
```

Comment:
TIL that `@here` and `@channel` _can_ be used...won't it lead to a lot of spam if someone uses this in a high-volume channel? :thinking:

Reply:
Slack doesn't have good permissions on that sort of thing :x

Reply:
/me sighs

Reply:
Update: I can't verify, its restricted to Slack owners, but that @here and @channel should be restricted to admins/owners only. So that guideline is strictly for admins.